samples/08.suggested-actions/adapter_with_error_handler.py

```diff
@@ -0,0 +1,46 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import sys
+from datetime import datetime
+
+from botbuilder.core import (
+    BotFrameworkAdapter,
+    BotFrameworkAdapterSettings,
+    TurnContext
+)
+from botbuilder.schema import ActivityTypes, Activity
+
+
+class AdapterWithErrorHandler(BotFrameworkAdapter):
+    def __init__(
+        self,
+        settings: BotFrameworkAdapterSettings
+    ):
+        super().__init__(settings)
+
+        # Catch-all for errors.
+        async def on_error(context: TurnContext, error: Exception):
```

Comment:
Please move this to main.py and add it directly to the instance like in the echo generator.